website/content/functional-database.md

```diff
@@ -0,0 +1,298 @@
+---
+title: Functional databases
```

Comment:
I'm wondering whether you want to reference things like [Functional Database Model](https://en.wikipedia.org/wiki/Functional_database_model) or [The Functional Database - Rich Hickey](https://www.youtube.com/watch?v=tRoVyblAGrs) and [Datomic](https://www.datomic.com/)?

Otherwise **A Functional Query Language** could be an alternative title?

Btw, I recall the Rich Hickey talk being very good but I've completely forgotten all the content now.

Reply:
Yes, Rich's talk is somewhat related, thanks for linking.

I'm open to changing the title, I'd want it be not only descriptive but also a bit provocative. "A Functional Query Language" may put off people that are fed up with new languages :D

Reply:
The title is completely up to you of course. I just felt that we're not really changing the functional nature of the database (as opposed to what Rich Hickey is talking about IIRC) but rather how you query or retrieve data.

* A functional paradigm for querying databases
* A functional approach to data queries
* ?

Reply:
That's true - it's not the database, but just the query language...